packages/imperative/src/config/cmd/edit/edit.definition.ts

```diff
@@ -24,7 +24,7 @@ export const editDefinition: ICommandDefinition = {
     summary: "edit config files",
     description: `Edit an existing config file in your system's default text editor.\n\n` +
         `In a graphical environment, the application associated with JSON files will be launched. ` +
-        `In a command-line environment, vi will be launched. To override the command-line editor used, specify it in ` +
+        `In a command-line environment, vi will be launched. To override your default editor, specify the chosen program's executable location by using ` +
```

Comment:
Thanks for revising to make the wording more clear 🙂

Instead of "the chosen program's executable location", would we want to say "the command to run the chosen program's executable"? It won't always be a full path to the executable location, for example `ZOWE_EDITOR=code` or `ZOWE_EDITOR=nano` are valid.

Reply:
can we merge both of these sentiments somehow? because yes you're right but you're also more knowledgeable than I am so that would make more sense to you than it would me

Reply:
Just offering yet another alternative for your consideration.

Change:
    ... specify the chosen program's executable location by using ...

To:
    ... specify the command to run your chosen editor by using ...